cmd/notary/main.go

```diff
@@ -77,10 +77,7 @@ func (n *notaryCommander) parseConfig() (*viper.Viper, error) {
 	// Get home directory for current user
 	homeDir, err := homedir.Dir()
 	if err != nil {
-		return nil, fmt.Errorf("cannot get current user home directory: %v", err)
```

Comment:
nit:  we should specify that the default notary directory instead will be `$(pwd)/.notary` maybe, as part of the warning, since we are still setting a default, it just may be in an unexpected place.

Reply:
Oh I see what you mean, can definitely change that :wink: